unlock-app/src/queries/locksByOwner.js

```diff
@@ -7,18 +7,9 @@ export default function locksByOwnerQuery() {
         id
         address
         name
-        tokenAddress
```

Comment:
This is not normalized so we cannot show it on the UI for now. (locks are loaded from alchemy anyway...) We just use The Graph has a "hint" until we have all the data we need in there.